src/bosh-director/spec/unit/problem_handlers/unresponsive_agent_spec.rb

```diff
@@ -1,346 +0,0 @@
-require 'spec_helper'
```

Comment:
I'm wandering why this change deletes the src/bosh-director/spec/unit/problem_handlers/unresponsive_agent_spec.rb?

Reply:
you are totally right (again). Removing that file looks like an accident and was reverted.

Reply:
thanks :-)